bundles/org.openhab.binding.bondhome/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -36,6 +36,13 @@
 		<category>Heating</category>
 	</channel-type>
 
+	<channel-type id="rawFanSpeedChannelType">
+		<item-type>Number</item-type>
+		<label>Raw Fan Speed</label>
+		<description>Sets fan speed using raw Bond values</description>
+		<category>Advanced</category>
```

Comment:
This category/icon doesn't exist, but could be:
```suggestion
		<category>Fan</category>
```

or same as for `fanSpeedChannelType`, if this is actually the same fan, i.e. `Heating`.

Reply:
I'll switch it to Heating to match the Dimmer interpreted fan speed